browser/help/topics/v4-browser-hts.md

```diff
@@ -0,0 +1,223 @@
+- `variant_id`: gnomAD variant ID.- `faf95_joint`: A struct containing joint (exome + genome) FAF information (95% CI).
+  - `grpmax`: Groupmax FAF value for all genetic ancestry groups across exomes + genomes.
+  - `grpmax_gen_anc`: Genetic ancestry group associated with the value `grpmax` above.
+- `faf99_joint`: A struct containing joint (exome + genome) FAF (99% CI). - `colocated_variants`: Array containing variants that are located at the same locus as this variant.
+- `coverage`: Struct containing coverage information for locus.
+  - `exome`: Struct containing exome coverage information.
+  - `mean`: Mean depth of coverage at this locus.
+  - `median`: Median depth of coverage at this locus.
+  - `over_1`: Percentage of samples with a coverage greater than 1 at this locus.
+  - `over_5`: TPercentage of samples with a coverage greater than 5 at this locus
+  - `over_10`: Percentage of samples with a coverage greater than 10 at this locus
+  - `over_15`: Percentage of samples with a coverage greater than 15 at this locus.
+  - `over_20`: Percentage of samples with a coverage greater than 20 at this locus
+  - `over_25`: Percentage of samples with a coverage greater than 25 at this locus.
+  - `over_30`: Percentage of samples with a coverage greater than 30 at this locus.
+  - `over_50`: Percentage of samples with a coverage greater than 50 at this locus.
+  - `over_100`: Percentage of samples with a coverage greater than 100 at this locus.
+  - `genome`: Struct containing genome coverage information. Contains the same fields as `exome` above.
+  - `transcript_consequences`: Array containing variant transcript consequence information.
+  - `biotype`: Transcript biotype.
+  - `consequence_terms`: Array of predicted functional consequences.
+  - `domains`: Sset containing protein domains affected by variant.
+  - `gene_id`: Unique ID of gene associated with transcript.
+  - `hgvsc`: HGVS coding sequence notation for variant.
+  - `hgvsp`: HGVS protein notation for variant.
+  - `is_canonical`: Whether transcript is the canonical transcript.
+  - `lof_filter`: Variant LoF filters (from [LOFTEE](https://github.com/konradjk/loftee)).
+  - `lof_flags`: LOFTEE flags.
+  - `lof`: Variant LOFTEE status (high confidence [HC] or low confidence [LC]).
+  - `major_consequence`: Primary consequence associated with transcript.
+  - `transcript_id`: Unique transcript ID.
+  - `transcript_version`: Transcript version.
+  - `gene_version`: Gene version.
+  - `is_mane_select`: Whether transcript is the MANE select transcript.
+  - `is_mane_select_version`: MANE Select version, has a value if this transcript is the MANE select transcript.
+  - `refseq_id`: RefSeq ID associated with transcript.
+  - `refseq_version`: RefSeq version.
+
+#### gnomAD v4.1. browser gene models Hail Table annotations
+
+Row fields:
```

Comment:
just checking that the globals are removed after full pipeline has been run -- I see this in `gs://gnomad-v4-data-pipeline/output/genes/genes_grch38_annotated_5.ht`:
```
----------------------------------------
Global fields:
    'annotations': struct {
        'canonical_transcript': struct {

        }
        'mane_select_transcript': struct {
            'version': str
        }
    }
----------------------------------------
```

Reply:
sorry, I worded that comment poorly -- it might be helpful to keep this global
```
- `mane_select_version`: MANE Select version used to annotate variants.
```
on this table as well (since this specifically contains gene information)